doc/nrf/config_and_build/configuring_app/sysbuild_images.rst

```diff
@@ -24,21 +32,21 @@ These options determine whether the secure boot image is included on the network
+|               ``SB_CONFIG_NETCORE_EMPTY``               | |NCS| empty network core image :zephyr_file:`samples/nrf5340/empty_net_core`                              |
 +---------------------------------------------------------+-----------------------------------------------------------------------------------------------------------+
-| :kconfig:option:`SB_CONFIG_NETCORE_HCI_IPC`             | Zephyr hci_ipc Bluetooth image :zephyr_file:`samples/bluetooth/hci_ipc`                                   |
+|               ``SB_CONFIG_NETCORE_HCI_IPC``             | Zephyr hci_ipc Bluetooth image :zephyr_file:`samples/bluetooth/hci_ipc`                                   |
 +---------------------------------------------------------+-----------------------------------------------------------------------------------------------------------+
-| :kconfig:option:`SB_CONFIG_NETCORE_RPC_HOST`            | |NCS| rpc_host Bluetooth image :zephyr_file:`samples/bluetooth/rpc_host`                                  |
+|               ``SB_CONFIG_NETCORE_RPC_HOST``            | |NCS| rpc_host Bluetooth image :zephyr_file:`samples/bluetooth/rpc_host`                                  |
 +---------------------------------------------------------+-----------------------------------------------------------------------------------------------------------+
-| :kconfig:option:`SB_CONFIG_NETCORE_802154_RPMSG`        | Zephyr 802.15.4 image :zephyr_file:`samples/boards/nrf/ieee802154/802154_rpmsg`                           |
+|               ``SB_CONFIG_NETCORE_802154_RPMSG``        | Zephyr 802.15.4 image :zephyr_file:`samples/boards/nrf/ieee802154/802154_rpmsg`                           |
 +---------------------------------------------------------+-----------------------------------------------------------------------------------------------------------+
-| :kconfig:option:`SB_CONFIG_NETCORE_MULTIPROTOCOL_RPMSG` | |NCS| multiprotocol_rpmsg Bluetooth and 802.15.4 image :zephyr_file:`samples/nrf5340/multiprotocol_rpmsg` |
+|               ``SB_CONFIG_NETCORE_MULTIPROTOCOL_RPMSG`` | |NCS| multiprotocol_rpmsg Bluetooth and 802.15.4 image :zephyr_file:`samples/nrf5340/multiprotocol_rpmsg` |
 +---------------------------------------------------------+-----------------------------------------------------------------------------------------------------------+
-| :kconfig:option:`SB_CONFIG_NETCORE_IPC_RADIO`           | |NCS| ipc_radio image :zephyr_file:`applications/ipc_radio`                                               |
+|               ``SB_CONFIG_NETCORE_IPC_RADIO``           | |NCS| ipc_radio image :zephyr_file:`applications/ipc_radio`                                               |
 +---------------------------------------------------------+-----------------------------------------------------------------------------------------------------------+
```

Comment:
ipc_radio, multiprotocol_rpmsk, hci_ipc, should be linking to the respective componants.